l10n_nl_tax_statement/views/l10n_nl_vat_statement_view.xml

```diff
@@ -0,0 +1,145 @@
+                    </div>
+                    <label for="name"/>
+                    <h1>
+                        <field name="name"/>
+                    </h1>
+                    <group>
+                        <group name="tax_report">
+                            <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
+                            <field name="parent_id" attrs="{'invisible': [('parent_id', '=', False)]}" groups="base.group_multi_company"/>
+                            <field name="display_multicompany_fiscal_unit" invisible="1" groups="base.group_multi_company"/>
+                            <field name="multicompany_fiscal_unit" attrs="{'readonly': [('state','in',['posted','final'])],'invisible': [('display_multicompany_fiscal_unit', '=', False)]}" groups="base.group_multi_company"/>
+                            <field name="fiscal_unit_company_ids" widget="many2many_tags" domain="[('parent_id', '=', company_id)]" attrs="{'readonly': [('state','in',['posted','final'])],'invisible': [('multicompany_fiscal_unit', '=', False)]}" groups="base.group_multi_company"/>
+                            <field name="date_range_id" attrs="{'readonly': [('state','in',['posted','final'])]}"/>
+                            <label for="from_date" string="Date"/>
+                            <div class="o_row">
+                                <field name="from_date" widget="daterange" nolabel="1" class="oe_inline" options="{'related_end_date': 'to_date'}" attrs="{'readonly': [('state','in',['posted','final'])]}"/>
+                                <i class="fa fa-long-arrow-right mx-2" aria-label="Arrow icon" title="Arrow"/>
+                                <field name="to_date" widget="daterange" nolabel="1" class="oe_inline" options="{'related_start_date': 'from_date'}" attrs="{'readonly': [('state','in',['posted','final'])]}"/>
+                            </div>
+                        </group>
+                        <group name="extra_parameters">
+                            <field name="date_posted"/>
+                            <field name="date_update"/>
+                            <field name="currency_id"/>
+                            <field name="target_move"/>
+                        </group>
+                    </group>
+                    <notebook name="notebook">
+                        <page name="statement" string="BTW Statement">
+                            <group attrs="{'invisible': [('parent_id','=',False)]}">
+                                <div>Check the statement of the parent Company.</div>
+                            </group>
+                            <group name="statement_lines" string="BTW Statement lines" attrs="{'invisible':['|',('state','!=','draft'),('parent_id','!=',False)]}">
```

Comment:
Why do we hide the lines of a parent statement is available? We can still show them so individual statements can be validated?

Reply:
To avoid duplication and subsequently confusion. See https://github.com/OCA/l10n-netherlands/issues/237. The parent is the only Tax Statement. The children are the ICPs.